VisualPinball.Engine/VPT/Table/Table.cs

```diff
@@ -35,7 +35,7 @@ public class Table : Item<TableData>, IRenderable
 		public readonly Dictionary<string, string> TableInfo = new Dictionary<string, string>();
 		public ITableResourceContainer<Texture> Textures = new DefaultTableResourceContainer<Texture>();
 		public ITableResourceContainer<Sound.Sound> Sounds = new DefaultTableResourceContainer<Sound.Sound>();
-		public readonly Dictionary<string, Collection.Collection> Collections = new Dictionary<string, Collection.Collection>();
+		public ITableResourceContainer<Collection.Collection> Collections = new DefaultTableResourceContainer<Collection.Collection>();

```

Comment:
In fact i finally used it because of Undo management (which was the main concern on Texture & Sound to not Record the full sidecar).
It has to be UnityEngine.Object at least, and CollectionData are not.

Reply:
It's fine then!